owasp-dependency-check-suppressions.xml

```diff
@@ -519,157 +175,184 @@
     <cve>CVE-2022-1278</cve>
   </suppress>
 
+
   <suppress>
-    <!-- Suppress aws-java-sdk-bundle cves -->
+    <!-- These are for elasticsearch in ranger soon to be fixed. -->
     <notes><![CDATA[
-    file name: aws-java-sdk-bundle-1.11.901.jar
+    file name: elasticsearch plugins
     ]]></notes>
-    <cve>CVE-2020-8570</cve>
-    <cve>CVE-2015-8559</cve>
-    <cve>CVE-2021-20291</cve>
-    <cve>CVE-2017-17485</cve>
-    <cve>CVE-2018-5968</cve>
-    <cve>CVE-2017-15095</cve>
-    <cve>CVE-2019-16942</cve>
-    <cve>CVE-2020-25649</cve>
-    <cve>CVE-2020-35491</cve>
-    <cve>CVE-2019-16943</cve>
-    <cve>CVE-2020-35490</cve>
-    <cve>CVE-2019-20330</cve>
-    <cve>CVE-2020-10673</cve>
-    <cve>CVE-2018-11307</cve>
-    <cve>CVE-2018-7489</cve>
-    <cve>CVE-2019-17267</cve>
-    <cve>CVE-2019-17531</cve>
-    <cve>CVE-2019-16335</cve>
-    <cve>CVE-2019-14893</cve>
-    <cve>CVE-2019-14540</cve>
-    <cve>CVE-2021-37136</cve>
-    <cve>CVE-2021-37137</cve>
+    <cve>CVE-2023-31417</cve>
+    <cve>CVE-2023-31418</cve>
+    <cve>CVE-2023-31419</cve>
   </suppress>
 
   <suppress>
-    <!-- Suppress hadoop-shaded-guava cves -->
     <notes><![CDATA[
-    file name: hadoop-shaded-guava-1.1.1.jar
+    file name: kafka-clients-3.2.0.jar
     ]]></notes>
-    <cve>CVE-2015-7430</cve>
-    <cve>CVE-2017-3162</cve>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.kafka/kafka\-clients@.*$</packageUrl>
+    <cve>CVE-2022-34917</cve>
+    <cve>CVE-2023-25194</cve>
   </suppress>
 
   <suppress>
-    <!-- False alarm for the Async javascript library (https://github.com/caolan/async) which is a dev dependency for the web console -->
     <notes><![CDATA[
-   file name: async-http-client-netty-utils-2.5.3.jar
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/org\.asynchttpclient/async-http-client-netty-utils@2.5.3$</packageUrl>
-    <cve>CVE-2021-43138</cve>
+    file name: woodstox-core-6.2.4.jar
+    ]]></notes>
+    <cve>CVE-2023-34411</cve>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+    file name: spatial4j-0.7.jar:
+    ]]></notes>
+    <cve>CVE-2014-125074</cve>
+  </suppress>
+
+  <suppress>
+    <notes><![CDATA[
+    file name: aws-sdk-bundle
+    ]]></notes>
+    <cve>CVE-2020-36518</cve>
+    <cve>CVE-2021-46877</cve>
+    <cve>CVE-2022-41881</cve>
```

Comment:
can suppression justification be added for these?

Reply:
those are addressed in https://github.com/apache/druid/pull/15443
I have added them here ( were missing before to pass the static checks)

Reply:
I will remove them after #15443 is merged